templates/data/kyma-integration-gardener-data.yaml

```diff
@@ -93,6 +107,22 @@ templates:
               CREDENTIALS_DIR: "/etc/credentials/kyma-tunas-prow-event-mesh"
             annotations:
               pipeline.platform: gardener_gcp
+          skr_aws_job_eventing:
+            labels:
+              preset-cluster-version: "true"
+            env:
+              KYMA_PROJECT_DIR: "/home/prow/go/src/github.com/kyma-project"
+              CREDENTIALS_DIR: "/etc/credentials/kyma-tunas-prow-event-mesh"
+              # KEB_PLAN_ID: "361c511f-f939-4621-b228-d0fb79a1fe15" # AWS
+              # KEB_PLAN_ID: "5cb3d976-b85c-42ea-a636-79cadda109a9" # Preview Plan
```

Comment:
Do we need those comments?

Reply:
it would be helpful for later if to switch to different plan.